live-examples/js-examples/map-prototype-set.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">var map1 = new Map();
+map1.set('bar', 'foo');
```

Comment:
Sorry @wbamberg - I'm not following - the second line sets. The gets show what  to expect when you query a set map and not.

Reply:
Sorry Mark, you are right of course.